examples/cellular/cellular.py

```diff
@@ -0,0 +1,304 @@
+    bus_el_ec_2_1 = buses.Bus(label="bus_el_ec_2_1")
+    bus_el_ec_2_2 = buses.Bus(label="bus_el_ec_2_2")
+
+    es.add(bus_el_es)
+    ec_1.add(bus_el_ec_1)
+    ec_2.add(bus_el_ec_2)
+    ec_1_1.add(bus_el_ec_1_1)
+    ec_1_2.add(bus_el_ec_1_2)
+    ec_2_1.add(bus_el_ec_2_1)
+    ec_2_2.add(bus_el_ec_2_2)
+
+    sink_el_ec_1 = cmp.Sink(
+        label="sink_el_ec_1",
+        inputs={bus_el_ec_1: flows.Flow(fix=demand_1, nominal_value=1)},
+    )
+    sink_el_ec_2 = cmp.Sink(
+        label="sink_el_ec_2",
+        inputs={bus_el_ec_2: flows.Flow(fix=demand_2, nominal_value=1)},
+    )
+    sink_el_ec_1_1 = cmp.Sink(
+        label="sink_el_ec_1_1",
+        inputs={bus_el_ec_1_1: flows.Flow(fix=demand_1_1, nominal_value=1)},
+    )
+    sink_el_ec_1_2 = cmp.Sink(
+        label="sink_el_ec_1_2",
+        inputs={bus_el_ec_1_2: flows.Flow(fix=demand_1_2, nominal_value=1)},
+    )
+    sink_el_ec_2_1 = cmp.Sink(
+        label="sink_el_ec_2_1",
+        inputs={bus_el_ec_2_1: flows.Flow(fix=demand_2_1, nominal_value=1)},
+    )
+    sink_el_ec_2_2 = cmp.Sink(
+        label="sink_el_ec_2_2",
+        inputs={bus_el_ec_2_2: flows.Flow(fix=demand_2_2, nominal_value=1)},
+    )
+
+    ec_1.add(sink_el_ec_1)
+    ec_2.add(sink_el_ec_2)
+    ec_1_1.add(sink_el_ec_1_1)
+    ec_1_2.add(sink_el_ec_1_2)
+    ec_2_1.add(sink_el_ec_2_1)
+    ec_2_2.add(sink_el_ec_2_2)
+
+    source_el_ec_1 = cmp.Source(
+        label="source_el_ec_1",
+        outputs={
+            bus_el_ec_1: flows.Flow(
+                max=pv_1, nominal_value=1, variable_costs=5
+            )
+        },
+    )
+    source_el_ec_2 = cmp.Source(
+        label="source_el_ec_2",
+        outputs={
+            bus_el_ec_2: flows.Flow(
+                max=pv_2, nominal_value=1, variable_costs=5
+            )
+        },
+    )
+    source_el_ec_1_1 = cmp.Source(
+        label="source_el_ec_1_1",
+        outputs={
+            bus_el_ec_1_1: flows.Flow(
+                max=pv_1_1, nominal_value=1, variable_costs=10
+            )
+        },
+    )
+    source_el_ec_1_2 = cmp.Source(
+        label="source_el_ec_1_2",
+        outputs={
+            bus_el_ec_1_2: flows.Flow(
+                max=pv_1_2, nominal_value=1, variable_costs=1
+            )
+        },
+    )
+    source_el_ec_2_1 = cmp.Source(
+        label="source_el_ec_2_1",
+        outputs={
+            bus_el_ec_2_1: flows.Flow(
+                max=pv_2_1, nominal_value=1, variable_costs=5
+            )
+        },
+    )
+    source_el_ec_2_2 = cmp.Source(
+        label="source_el_ec_2_2",
+        outputs={
+            bus_el_ec_2_2: flows.Flow(
+                max=pv_2_2, nominal_value=1, variable_costs=5
+            )
+        },
+    )
+
+    ec_1.add(source_el_ec_1)
+    ec_2.add(source_el_ec_2)
+    ec_1_1.add(source_el_ec_1_1)
+    ec_1_2.add(source_el_ec_1_2)
+    ec_2_1.add(source_el_ec_2_1)
+    ec_2_2.add(source_el_ec_2_2)
+
+    connector_el_ec_1 = buses.Bus(
```

Comment:
It might be cleaner to use the ’Link’ here, as it does not allow the flow to return into its original cell. (It still can if the loop is closed in the other cell.)